### PR TITLE
MM-54743 - Fix: In compact mode, the call card displays inline

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- Had to do some trickery to force the display to wrap (because we don't control our custom post's container).
- Doesn't affect non-compact mode, or when a call title is present. If you can think of any other situations I didn't think, please let me know.

| compact, full width | compact, reduced width |
|--|--|
| image | image |

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-54743

### CODE REVIEW

File: webapp/src/components/custom_post_types/post_type/component.tsx [hunk 1]
```diff
@@ -136,6 +138,8 @@ const PostType = ({
         );
     }
 
+    const compactTitle = compactDisplay ? '\u200b' : '';
```
Comment (streamer45) [inline webapp/src/components/custom_post_types/post_type/component.tsx:141]: Any reason for not using something more semantically defined like the `<br>` tag?

**Review by streamer45 — APPROVED** [review]

**Review by matthewbirtch — CHANGES_REQUESTED** [review]
@cpoile this looks good, but I did notice an issue in the RHS in compact mode. There seems to be extra space added that is not there in standard display mode:

**Compact display mode**
Notice the extra space between the username and the call card
image

**Standard display mode**
image

Other than that, this should be good.

**Comment by cpoile** [comment]
Fixed now, thanks for pointing that out Matt!

**Review by matthewbirtch — APPROVED** [review]
Looks good. Thanks Chris!

**Pull request merged**